sabertooth_driver_node:
  ros__parameters:
    serial_port: "/dev/ttyACM0"
    feedback_rate_hz: 10.0